package.json

```diff
@@ -12,6 +12,7 @@
     "prop-types": "^15.8.1",
```

Comment:
- [ ] Please add the title of the project in the Readme file 
![image](https://user-images.githubusercontent.com/74744787/218261471-062ce4b0-9d6f-4482-9a97-d9c6dfe0f3d6.png)